oio/conscience/stats/http.py

```diff
@@ -71,8 +71,6 @@ def get_stats(self):
                 result = self._parse_func(resp.read())
             else:
                 raise Exception("status code != 200: %s" % resp.status)
-        except Exception as e:
-            self.logger.debug("get_stats error: %s", e)
```

Comment:
The return statement in the `finally` block will prevent any exception from being raised from this method. It should be moved here.